package.json

```diff
@@ -3,9 +3,6 @@
   "version": "3.7.2",
   "description": "Datatables for React using Material-UI",
   "main": "dist/index.js",
-  "files": [
-    "dist"
-  ],
```

Comment:
What's the reason?

Reply:
Sorry I accidentally pushed an update to this branch when I was testing something else.